api/config/feature-toggles-config.js

```diff
@@ -74,4 +74,11 @@ export default {
     devDefaultValues: { test: true, reviewApp: true },
     tags: ['frontend', 'team-prescription'],
   },
+  usePixOrgaNewAuthDesign: {
+    type: 'boolean',
+    description: 'Displays the new design of authentication pages',
+    defaultValue: false,
+    devDefaultValues: { test: false, reviewApp: true },
```

Comment:
On ne veut pas l'activer sur les review apps pour le moment.
```suggestion
    devDefaultValues: { test: false, reviewApp: false },
```